lib/src/command/global_run.dart

```diff
@@ -27,6 +28,9 @@ class GlobalRunCommand extends PubCommand {
   GlobalRunCommand() {
     argParser.addFlag('enable-asserts', help: 'Enable assert statements.');
     argParser.addFlag('checked', abbr: 'c', hide: true);
+    argParser.addMultiOption('enable-experiment',
+        help: 'Runs the executable in a VM with the given experiments enabled.',
```

Comment:
1) Wondering if we should include details saying that the target executable won't be precompiled, so it'll start up more slowly
2) Is there somewhere  we can point folks to get more info on experiments?

https://github.com/dart-lang/sdk/blob/master/docs/process/experimental-flags.md is not super helpful

CC @mit-mit

Reply:
Og, maybe we should add a short page on dart.dev about experiment flags, WDYT @kwalrath ?

Reply:
Well, I think linking to the github repo is fine – maybe we could have a redirect like flutter.dev/go/... right?

Reply:
Sure, we could have a page on dart.dev. Where would it go? What should it cover?

Reply:
@kwalrath i sent some notes in email